.build.yml

```diff
@@ -90,32 +72,14 @@ variables:
     -enable-checker security.insecureAPI.strcpy"'
 
 before_scripts:
```

Comment:
seems this line `"before_scripts:"` can be removed too

Reply:
It doesn't hurt but OK :)